### PR TITLE
Created initial Bean Class API 

### PR DESCRIPTION
**Pull request by LiamTalberg**

### CODE REVIEW

File: src/main/java/com/beantastic/api/controllers/BeanClassController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class BeanClassController {
+
+    private final BeanClassRepository beanClassRepository;
+
+    public BeanClassController(BeanClassRepository beanClassRepository) {
+        this.beanClassRepository = beanClassRepository;
+
+    }
+
+    @GetMapping("/getBeanClasses")
```
Comment (CursoryWaif32BBD) [inline src/main/java/com/beantastic/api/controllers/BeanClassController.java:19]: can it be just /beanClasses

File: src/main/java/com/beantastic/api/models/entities/BeanClassStatistic.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@Data
+@Entity
+@Table
+public class BeanClassStatistic {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int beanClassStatisticsId;
+
+    @ManyToOne
+    private BeanClass beanClass;
```
Comment (Liam-Quick-Vodacom) [inline src/main/java/com/beantastic/api/models/entities/BeanClassStatistic.java:15]: I think it makes sense to have this relationship be on the beanclass side

Comment (CursoryWaif32BBD) [reply src/main/java/com/beantastic/api/models/entities/BeanClassStatistic.java:14]: still seeing it here

**Review by CursoryWaif32BBD — APPROVED** [review]
looks good, just think the bean class should own the 'List' of statistics not the other way around

**Pull request merged**